pipeline/article_scraper/processor.py

```diff
@@ -27,18 +27,18 @@
 
 process.start()
 
-json_file = data_dir + 'blog_articles_{0}.jsonl'.format(
+json_file = data_dir + '/blog_articles_{0}.jsonl'.format(
     time.strftime("%d_%m_%Y"))
 if os.path.isfile(json_file):
-    flattened_article_df = pd.read_json(json_file)
+    flattened_article_df = pd.read_json(json_file, lines=True)
 
 # Save output file
-field_type = {'source': sqlalchemy.types.NVARCHAR(length=255),
-              'title': sqlalchemy.types.NVARCHAR(length=255),
+field_type = {'source': sqlalchemy.types.Unicode,
```

Comment:
Please specify the length of the `Unicode` as in `NVARCHAR`. You might want to use *sqlalchemy.types.UnicodeText* for articles, since `Unicode` and `NVARCHAR` are limited to a certain text length.